src/storage/hummock_sdk/src/lib.rs

```diff
@@ -281,7 +281,7 @@ impl EpochWithGap {
     pub fn new(epoch: u64, spill_offset: u16) -> Self {
         #[cfg(not(feature = "enable_test_epoch"))]
         {
-            debug_assert_eq!(epoch & EPOCH_MASK, 0);
+            debug_assert!(((epoch & EPOCH_MASK) == 0) || (epoch == HummockEpoch::MAX));
```

Comment:
Can you add some docs to describe the background here?

Reply:
> how about replace epoch == HummockEpoch::MAX with function is_compatibility_max_epoch

+1, we may have 256 available max epoch, epochs in (MAX_EPOCH, u64 max)  should pass this check.